aws/identity/lib/config.ts

```diff
@@ -0,0 +1,30 @@
+import * as pulumi from "@pulumi/pulumi";
+
+const config = new pulumi.Config();
+
+//
+// Define groups.
+//
+
+export const defineAdminsGroup = config.get("admins ") || "true";
```

Comment:
Several of these have an extra space after the string in `.get()`. I'm guessing that was unintentional?